apps/consumers.py

```diff
@@ -13,28 +13,29 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-
 from string import Template
 
 from channels.db import database_sync_to_async
 from enumfields import Enum
-from rest_framework_json_api.renderers import JSONRenderer
+from rest_framework_json_api.renderers import JSONRenderer as JSON_VNDRenderer
```

Comment:
'vnd' stands for 'vendor' which isn't really descriptive. Also it's against python style to have underscores in class names. Use JSONAPIRenderer instead